### PR TITLE
Add setup instructions

### PR DESCRIPTION
**Pull request by libronlopes**
add file for wiki setup instuctions

### CODE REVIEW

**Review by jdknives — COMMENTED** [review]
This is suboptimal. Most users will use the GIthub interface and therefore we need to put these instructions in the README as the README is automatically rendered.

**Comment by libronlopes** [comment]
> 
> 
> This is suboptimal. Most users will use the GIthub interface and therefore we need to put these instructions in the README as the README is automatically rendered.

Okay, ill change the file name to README

File: README.md [hunk 1]
```diff
@@ -17,3 +17,35 @@ Please keep the folder structure organized. <br>
 
 If you want to contribute to this wiki please have a look at [Contribute](https://skycoin.github.io/wiki/contribute/).
```
Comment (jdknives) [inline README.md:18]: This sentence should be removed as we specifically show the setup instructions below.

**Review by jdknives — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (libronlopes) [reply README.md:18]: Okay.

**Pull request merged**